### ISSUE TITLE
List of potentially useful data sets

### ISSUE THREAD

**Issue by SylvainTakerkart** [INCF/neuroscience-data-structure#7]
Let us list here the data sets that could be used for this standardization initiative...

**Comment by SylvainTakerkart** [comment]
from @satra in this post https://github.com/INCF/neuroscience-data-structure/issues/4#issuecomment-741804263:

you can look at the datasets on https://dandiarchive.org (there are about 38 dandisets covering 4 species, intra/extracellular and optical recordings)

**Comment by SylvainTakerkart** [comment]
from @jmbudd in this post: https://github.com/INCF/neuroscience-data-structure/issues/4#issuecomment-741459222

> The Buzsaki lab, for example, has for many years shared their rodent hippocampal recording datasets many of which are available at crcns.org. In the process they have created a custom (session-based) data structure for their increasingly complicated combined ephys and behavioural experiments (see https://buzsakilab.com/wp/data-structure-and-format/).

**Comment by SylvainTakerkart** [comment]
from @SylvainTakerkart in this post: https://github.com/INCF/neuroscience-data-structure/issues/4#issuecomment-741832906

> This dataset (utah array ephys, several sessions, several animals, exhaustive metadata) could also be useful:
> https://doi.org/10.1038/sdata.2018.55
> https://doi.gin.g-node.org/10.12751/g-node.f83565/

**Comment by tgbugs** [comment]
A general pointer to the published sparc datasets, I'll pull out the stranger ones later https://sparc.science/data?type=dataset.

**Comment by JuliaSprenger** [comment]
> from @SylvainTakerkart in this post: [#4 (comment)](https://github.com/INCF/neuroscience-data-structure/issues/4#issuecomment-741832906)
> 
> > This dataset (utah array ephys, several sessions, several animals, exhaustive metadata) could also be useful:
> > https://doi.org/10.1038/sdata.2018.55
> > https://doi.gin.g-node.org/10.12751/g-node.f83565/

There exists also an attempt to restructure this dataset according to [AnDO](https://int-nit.github.io/AnDOChecker/) specifications here: https://gin.g-node.org/sprenger/multielectrode_grasp/src/ando

**Comment by chrisvdt** [comment]
I would like to upload some empty datasets, but I can not push changes at the moment.
For example I have an empty dataset for this study I would like to deposit here; Shape perception via a high-channel-count neuroprosthesis in monkey visual cortex. Chen X, Wang F, Fernandez E, Roelfsema PR. Science. 2020 Dec4;370(6521):1191-1196. doi: 10.1126/science.abd7435. PMID: 33273097.

**Comment by Remi-Gau** [comment]
I have told @SylvainTakerkart  and @JuliaSprenger but I figured that I might as well put this here, but I am "stress testing" the BIDS  BEP with an old dataset of mine that has both single cell recording and microscopy data:

https://gin.g-node.org/RemiGau/5HT_pain_ephys_raw

**Comment by SylvainTakerkart** [comment]
> https://gin.g-node.org/RemiGau/5HT_pain_ephys_raw

Thanks @Remi-Gau ! But for info, I get a 404 error on this link...

Also, one question: is this a joint stress-test for BEP031 and BEP032?

**Comment by Remi-Gau** [comment]
> > https://gin.g-node.org/RemiGau/5HT_pain_ephys_raw
> 
> Thanks @Remi-Gau ! But for info, I get a 404 error on this link...

Had forgotten to make the repo public :facepalm: 


> Also, one question: is this a joint stress-test for BEP031 and BEP032?
Yup it is: so some files already in there relate to the microscopy aspect.

If this is a problem for readability, let me know and I can try to "split things" while we iron the details of each BEP.

**Comment by yarikoptic** [comment]
> from @satra in this post [#4 (comment)](https://github.com/INCF/neuroscience-data-structure/issues/4#issuecomment-741804263):
> 
> > you can look at the datasets on https://dandiarchive.org (there are about 38 dandisets covering 4 species, intra/extracellular and optical recordings)

note that since then we have them all exposed also as DataLad datasets: https://github.com/dandi/dandisets

**Comment by SylvainTakerkart** [comment]
> > from @satra in this post [#4 (comment)](https://github.com/INCF/neuroscience-data-structure/issues/4#issuecomment-741804263):
> > > you can look at the datasets on https://dandiarchive.org (there are about 38 dandisets covering 4 species, intra/extracellular and optical recordings)
> 
> note that since then we have them all exposed also as DataLad datasets: https://github.com/dandi/dandisets

yes, clearly! we'd need manpower to try to convert some of them (selected to be interesting from some point of view to be discussed) to BIDS... any idea @satra @yarikoptic ?
